### ISSUE TITLE
External link indicator for NRRD markdown

### ISSUE THREAD

**Issue by cthomasONRR** [DOI-ONRR/nrrd#2941]
We want to make sure all external links in NRRD have an [external link indicator](https://fontawesome.com/v5/icons/external-link?f=classic&s=solid), but still open in the same window like they do now.

- Search web to see if there is a simple markdown command I can add to the link code.
-  If there is, I will work to implement it on all external links
-  If I cannot fix myself, I will get Jeff's help

**Comment by cthomasONRR** [comment]
HTML option: https://github.com/PsyTeachR/book-template/issues/3
`<a href="google.com">the link</a>`


This says it's for R, but might work too https://psyteachr.github.io/book-template/psyteachr-style-guide.html#links
`[link text](url){target="_blank"} `

From old onrr.gov:
`<a href="URL" target="_blank" >link text</a>.`

**Comment by cthomasONRR** [comment]
@greensomes 

I tried several code additions in PR https://github.com/DOI-ONRR/nrrd/pull/2945, but none have worked.

I think we might need to install the font package (or something similar) to what's listed here: https://www.w3schools.com/icons/tryit.asp?filename=tryicons_fa-external-link

I'll move this to the product backlog until we have more time to work on it.

**Comment by cthomasONRR** [comment]
Recording my PR attempts here in case we decide to delete it:

`<a href="google.com">the link</a>`
Does not work- no indicator but still opens in same window
![image](https://github.com/DOI-ONRR/nrrd/assets/72526208/dcfe131c-c878-4e57-b578-dfbfd9859d78)

`[link text](url){target="_blank"} `
Does not work- "{target="_blank"}" shows in text. Link still opens in same window.
![image](https://github.com/DOI-ONRR/nrrd/assets/72526208/83a627a9-5dc1-43ae-ac29-f76260808388)

`<a href="URL" target="_blank" >link text</a>.`
Does not work- no indicator and link now opens in new tab.
This is what we used in the old onrr.gov but it's not working for MDX.
![image](https://github.com/DOI-ONRR/nrrd/assets/72526208/edb8a807-258d-4548-9536-3c4c633289d8)

`<ExternalLink href="URL">link text</ExternalLink>`
Does not work either.
![image](https://github.com/DOI-ONRR/nrrd/assets/72526208/a9036fac-2b8d-4455-8d43-530f47500414)


I think we need to install some sort of theme package? 
https://kabartolo.github.io/chicago-docs-demo/docs/components-and-hooks/#externallink

This is what USWDS shows 
https://designsystem.digital.gov/components/link/ 
![image](https://github.com/DOI-ONRR/nrrd/assets/72526208/1e1a9f08-7dea-40fe-8316-7727e0876b4a)

I tried this and it does not work. No external link icon for "Federal lands" and adding the new code for Federal lands somehow messed up the OCS link.
![image](https://github.com/DOI-ONRR/nrrd/assets/72526208/256fe87d-d6a1-4aa7-ad53-a9caccc69665)

I tried the code here: https://www.w3schools.com/icons/tryit.asp?filename=tryicons_fa-external-link
by just adding it to the end of the markdown format:
`[link text](link url)<i class="fa fa-external-link" style="font-size:24px"></i> a`
And it didn't do anything:
![image](https://github.com/DOI-ONRR/nrrd/assets/72526208/cfc9656e-6a57-4f6b-9b3c-6d8ef3194d8c)

I think we need to install the font package (or something similar) to what's listed here https://www.w3schools.com/icons/tryit.asp?filename=tryicons_fa-external-link
https://fontawesome.com/v5/icons/external-link?f=classic&s=solid

**Comment by greensomes** [comment]
Hi @cthomasONRR. We can use the [Material UI icons](https://v4.mui.com/api/icon/) currently available to us. I should be able to tackle this in the next sprint.

**Comment by greensomes** [comment]
I've added external link indicator icons to external links under the Downloads and How does revenue work pages.
